ops/logging.go

```diff
@@ -121,6 +119,14 @@ var loggedContentTypes = map[string]struct{}{
 	"application/vnd.kafka.v2+json":      {},
 }
 
+var loggedHeaders = []string{
+	"Content-Type",
+	"Content-Encoding",
+	"X-Forwarded-For",
+	"X-Correlation-ID",
+	"traceparent",
+}
```

Comment:
Anything else that would be useful to add?

Reply:
Perhaps it's time to have a strict warning about debug logging in the operational instructions. At least for high load environments

Reply:
Yeah, but that should go without saying though 😆 But sure, I'll add something to the docs